zb-db/src/main/java/io/zeebe/db/impl/rocksdb/transaction/ZeebeTransactionDb.java

```diff
@@ -449,15 +452,22 @@ public void createSnapshot(File snapshotDir) {
 
   @Override
   public void close() {
+    // Correct order of closing
+    // 1. transaction
+    // 2. options
+    // 3. column family handles
+    // 4. database
+    // 5. db options
+    // 6. column family options
+    // https://github.com/facebook/rocksdb/wiki/RocksJava-Basics#opening-a-database-with-column-families
+    Collections.reverse(closables);
     closables.forEach(
         closable -> {
           try {
             closable.close();
           } catch (Exception e) {
-            throw new RuntimeException(e);
+            // ignore to close others
```

Comment:
I would prefer to at least log the exception, even if we ignore them it might be an useful information